app/components/publications/publication-sidebar.hbs

```diff
@@ -36,31 +37,65 @@
               </div>
             {{/if}}
           </div>
+
+          {{!-- Status --}}
           <Publications::PublicationStatusSelector
                   @publicationStatus={{this.publicationStatus}}
                   @onChange={{this.setPublicationStatus}}
+                  @disabled={{or this.publicationStatus.isPublished this.publicationStatus.isWithdrawn}}
           />
-          <div class="auk-form-group">
-            <WebComponents::AuLabel for="blockInput">{{t "regulation-type"}}</WebComponents::AuLabel>
-            <WebComponents::AuPowerSelect @options={{this.regulationTypes}}
-                                          @isSearchable={{false}}
-                                          @onchange={{this.setRegulationType}}
-                                          @selected={{@publicationFlow.regulationType}}
-            />
-          </div>
-          <div class="auk-form-group">
-            <WebComponents::AuLabel for="blockInput">{{t "kind-of-publication"}}</WebComponents::AuLabel>
-            <WebComponents::AuPowerSelect @options={{this.publicationModes}}
-                                          @isSearchable={{false}}
-                                          @onchange={{this.setPublicationMode}}
-                                          @selected={{this.publicationFlow.mode}}
-            />
-          </div>
-          <div class="auk-form-group">
-            <Publications::UrgencyLevelCheckbox
-                    @urgencyLevel={{this.publicationFlow.urgencyLevel}}
-                    @onChange={{this.setUrgencyLevel}} />
-          </div>
+
+          {{!-- TODO: do the same for isPaused --}}
```

Comment:
TODO moet nog in dit ticket gebeuren of ?

Reply:
Voor pauzeerdatum is momenteel geen data-element voorzien. Dat moeten we verder buiten deze PR bekijken.